gcp-native-ts-k8s-ruby-on-rails-postgresql/app/start

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+set -e
+# Spawn the Rails App.
+echo "* SPAWNING RAILS:"
+bundle exec rails s -p 3000 -b '0.0.0.0' &
+
+# Wait for it to come up, and then ensure the database is created and migrated.
+sleep 5
+echo "* CREATING DATABASE (first time only):"
+bundle exec rails db:create
+echo "* MIGRATING DATABASE (if needed):"
+bundle exec rails db:migrate
+
+echo "* RUNNING..."
+wait
+
+echo "* Deleting DATABASE"
```

Comment:
This is a hack and probably doesn't work. We can't delete the db user until the DB created using it is deleted. It probably makes better sense to move to container lifecycle hook instead.

Reply:
I created the DB in pulumi instead which makes the destroy work better.